src/utils/Multicallable.huff

```diff
@@ -0,0 +1,106 @@
+// Calldata
+#define constant DATA_LEN = 0x24
+#define constant DATA_OFFSET = 0x44
+// Memory
+#define constant RES = 0x20
+#define constant RES_OFF = 0x40
+
+/// @notice Macro that enables a single call to call multiple methods within its contract.
+/// @author Solady (https://github.com/vectorized/solady/blob/main/src/utils/Multicallable.sol)
+/// @author Modified from Solmate (https://github.com/transmissions11/solmate/blob/main/src/utils/Multicallable.sol)
+/// @author clabby <https://github.com/clabby>
+#define macro MULTICALL() = takes (0) returns (0) {
```

Comment:
Should we conform to the natspec at top of contract? (we should probably also add to huff-docs style guide fwiw)

Reply:
Yeah- I think that's a good idea. Will throw up an issue in the docs, fixed it :smile: